BlazarUI/app/scripts/components/sidebar/BuildsSidebarContainer.jsx

```diff
@@ -56,17 +57,30 @@ class BuildsSidebarContainer extends Component {
   }
 
   render() {
+    let miniLogo;
+    if (this.props.isCollapsed) {
+      miniLogo = <SidebarLogo mini={true} />
+    }
     // BuildsNotifier.updateModules(this.state.builds.modules);
     return (
-      <Sidebar>
-        <BuildsSidebar
-          builds={this.state.builds}
-          stars={this.state.stars}
-          loading={this.state.loading} 
-          persistStarChange={this.persistStarChange} />
-      </Sidebar>
+      <div>
+        {miniLogo}
```

Comment:
You're right, `undefined` totally works but it's more explicit to check for the empty case and return `null` instead of letting implicit `undefined` returns do it for you. Definitely more of a stylistic thing but explicit > implicit.